src/common/options/mgr.yaml.in

```diff
@@ -5,8 +5,8 @@ options:
 - name: mgr_data
   type: str
   level: advanced
-  desc: Filesystem path to the ceph-mgr data directory, used to contain keyring.
-  fmt_desc: Path to load daemon data (such as keyring)
+  desc: Filesystem path to the Manager's ``ceph-mgr`` data directory, which contains keyrings
```

Comment:
in YAML, the backstick "\`" is reserved. see https://yaml.org/spec/1.2.2/#rule-c-reserved
if we really need to include the reserved indicators like `@` or "\`". probably we can use the double-quoted style scalar by quoting the string with double quotes.

but please note the `desc` field is not only rendered by sphinx if both `fmt_desc` and `long_desc` fields are missing, it is also used directly when printing options with `ceph config dump` command. in general, "desc" should be a short description of the option. while "long_desc" is complete sentences. it's encouraged to use `fmt_desc` for the reStructuredText formatted text. see https://docs.ceph.com/en/latest/dev/config/#defining-config-options